baseband/mark5b/base.py

```diff
@@ -64,6 +92,40 @@ def read_frame(self):
                                     ref_time=self.ref_time, nchan=self.nchan,
                                     bps=self.bps)
 
+    def get_frame_rate(self):
+        """Returns the number of frames per second.
+
+        Returns
+        -------
+        frame_rate : `~astropy.units.Quantity`
+            Frames per second.
+
+        Notes
+        -----
+        This function first tries to determine the frame rate by looking for
```

Comment:
"This function" -> "This method" (already corrected under #201, so hoping it makes it to here once we merge).